Core/Server/Request.php

```diff
@@ -58,61 +58,57 @@ private function _setup() {
 
             foreach ($_SERVER as $name => $value)
             {
+                if (strpos($name, 'HTTP_') === 0) {
+                    $name = strtolower(str_replace('_', '-', substr($name, 5)));
 
-                if (strpos($name,'HTTP_') !== false) {
```

Comment:
It's more accurate to compare `strpos()` with 0 because if it just not `false` it can be not at the beginning.